metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -165,6 +167,13 @@ func (sink *StackdriverSink) processMetrics(metricValues map[string]core.MetricV
 }
 
 func (sink *StackdriverSink) ExportData(dataBatch *core.DataBatch) {
+	// Make sure we don't export metrics too often.
+	if dataBatch.Timestamp.Before(sink.lastExportTime.Add(sink.minInterval)) {
+		glog.Infof("Skipping batch from %s because there hasn't passed %s from last export time %s", dataBatch.Timestamp, sink.minInterval, sink.lastExportTime)
```

Comment:
Time, Duration has string method defined. It's well formated:
```
I0809 09:21:05.043400       1 stackdriver.go:175] Skipping batch from 2017-08-09 09:21:00 +0000 UTC because there hasn't passed 1m40s from last export time 2017-08-09 09:20:00 +0000 UTC
```